services/slack.py

```diff
@@ -28,7 +28,7 @@ def post(text, channel):
         client.chat_postMessage(channel=channel, text=text)
     except SlackApiError:
         # failing to slack is never fatal, so log and do not error
-        logger.exception("Failed to notify slack")
+        logger.exception("Failed to notify slack", channel=channel)
```

Comment:
will this show up in the log output?

Reply:
I guess it will because of our log config, but I'd be inclined to include it in the exception message text?

Reply:
`logger` is a structlog logger here so it'll appear in our logs as `channel=some_string`. I'm not sure what we gain by duplicating it in the log message?

Reply:
It's disjointed - the channel will be at the top of the traceback, and the message at the bottom.

Also, in sentry, the log lines are separate from the exception. I think both is good: in the exception message for ease of use/obviousness, and in the log message for grepping/filtering